app/controllers/ops_controller/settings/automate_schedules.rb

```diff
@@ -17,10 +17,24 @@ def automate_schedules_set_vars
     }
   end
 
+  def name_included?(klass)
+    klass.safe_constantize.column_names.include?('name')
+  end
+
+  def columns_for_klass(klass)
+    if klass == 'Tenant'
+      [:name, :use_config_for_attributes]
+    elsif name_included?(klass)
+      [:name]
+    else
+      [:description]
+    end
+  end
+
   def fetch_target_ids
     if params[:target_class] && params[:target_class] != 'null'
-      targets = Rbac.filtered(params[:target_class]).select(:id, :name)
-      unless targets.nil?
+      targets = Rbac.filtered(params[:target_class]).select(:id, *columns_for_klass(params[:target_class]))
+      if targets.present?
         targets = targets.sort_by { |t| t.name.downcase }.collect { |t| [t.name, t.id.to_s] }
```

Comment:
Is it possible to have the sorting in the SQL with `order`? Isn't the `select` here an SQL-select?

Reply:
If you think about using `order` instead of `sort_by`, then... I tried:
```
[11] pry(#<OpsController>)> targets.order(:name)
NoMethodError: undefined method `order' for #<Array:0x007fe435b20310>
from (pry):9:in `fetch_target_ids'
```
What am I doing wrong? Thank you.

Reply:
@skateman it is possible but I think we should do it in follow up PR as it is not subject of this fix.